api/cache/clear.js

```
@@ -0,0 +1,340 @@
+          result.operations.push({
+            type: "namespace_clear",
+            namespace,
+            cleared: result.clearedCount,
+          });
+        }
+        result.success = true;
+        break;
+
+      case "selective":
+        // Selective clearing based on cache type
+        const operations = [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)